core/templates/dev/head/domain/opportunity/ExplorationOpportunitySummaryObjectFactorySpec.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for ExplorationOpportunitySummaryObjectFactory.
+ */
+
+import { ExplorationOpportunitySummaryObjectFactory } from
+  'domain/opportunity/ExplorationOpportunitySummaryObjectFactory';
+
+fdescribe('Exploration opportunity summary object factory', () => {
```

Comment:
This is still using `fdescribe`

Reply:
I missed this, thanks for the comment!
I was expecting lint to fail for this, is it failing for a similar reason? IF so why there isn't any clear log for the same?

Reply:
I've created an issue at https://github.com/oppia/oppia/issues/8068. (Marked `important`)

Reply:
Done!